docs/admin_guide.md

```diff
@@ -6,6 +6,8 @@ This reference contains descriptions of all the configuration commands, and deta
 
 The [Open Diffix website](https://www.open-diffix.org) has general information about the Open Diffix project. The [website FAQ](https://www.open-diffix.org/faq) is a good starting point. [This article](https://www.open-diffix.org/blog/diffix-elm-automates-what-statistics-offices-have-been-doing-for-decades) is a high-level overview of the anonymization mechanisms used by Diffix (version Elm). [This paper](https://arxiv.org/abs/2201.04351) is a complete specification and privacy analysis of Diffix Elm.
 
+> Unless otherwise stated, `master` branch documents Diffix Fir being the next, work-in-progress version.
```

Comment:
"This document applies to the Fir version of Diffix, which is currently a work in progress."

Reply:
sounds good x2